### PR TITLE
Treat bcmcmd as a supervisor task so we could collect stdout/stderr

### PR DESCRIPTION
**Pull request by qiluo-msft**
Also clean the unix socket file before dsserve

### CODE REVIEW

File: platform/broadcom/docker-syncd-brcm/start.sh [hunk 1]
```diff
@@ -33,5 +34,5 @@ wait_syncd() {
 # If this platform has an initialization file for the Broadcom LED microprocessor, load it
 if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
     wait_syncd
-    /usr/bin/bcmcmd -t 60 "rcload ${PLATFORM_DIR}/led_proc_init.soc"
+    supervisorctl start bcmcmd
```
Comment (lguohan) [inline platform/broadcom/docker-syncd-brcm/start.sh]: change the task name to ledinit

File: platform/broadcom/docker-syncd-brcm/start.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (jleveque) [inline platform/broadcom/docker-syncd-brcm/start.sh]: Reason for changing? Using `#!/usr/bin/env bash` will work even if the bash installation directory ever changes.

File: platform/broadcom/docker-syncd-brcm/start.sh (hunk 1 above)
Comment (jleveque) [reply platform/broadcom/docker-syncd-brcm/start.sh]: +1. This way we can differentiate other tasks run with bcmcmd, if we ever add any.

**Comment by jleveque** [comment]
Retest this please.

**Pull request merged**